teams_locations.go

```diff
@@ -35,6 +35,7 @@ type TeamsLocation struct {
 	AnonymizedLogsEnabled bool                   `json:"anonymized_logs_enabled"`
 	IPv4Destination       string                 `json:"ipv4_destination"`
 	ClientDefault         bool                   `json:"client_default"`
+	ECSSupport            bool                   `json:"ecs_support,omitempty"`
```

Comment:
```suggestion
	ECSSupport            *bool                   `json:"ecs_support,omitempty"`
```

https://github.com/cloudflare/cloudflare-go/blob/master/docs/conventions.md#booleans

Reply:
Updated